### PR TITLE
Implemented Screenshot Capture

### LINKED ISSUE #3: Screenshot Capture
# Objective
Capture screenshots every 100ms and save them to device storage without taking permission. Reference UxCam/UserExperior for implementation.

# Requirements

- Implement a mechanism to capture screenshots every 100ms.
- Save the captured screenshots to device storage.
- Ensure the process does not require explicit permission from the user.

# Steps to Implement

1. Create a ScreenshotCapture class.
2. Use a timer to trigger screenshot capture every 100ms.
3. Save the screenshots to a predefined directory on the device.
4. Ensure the functionality works on both Android and iOS.

# Resources
[Screenshot package](https://pub.dev/packages/screenshot)

### PR DESCRIPTION
**Pull request by Saumya-28**
Closes #3 
## Implemented Screenshot Capture
LogsuiteScreenshot

### CODE REVIEW

**Review by avinashkranjan — CHANGES_REQUESTED** [review]
When I've already mentioned in the Issue `Capture screenshots every 100ms and save them to device storage without taking permission` why you taking permission and handling it, create a solution where you can save the screenshots with permission of `EXTERNAL_STORAGE` from the user.

**Comment by Saumya-28** [comment]
@avinashkranjan I have updated the code , kindly check and let me know.

File: example/android/app/src/main/AndroidManifest.xml
Comment (avinashkranjan) [inline example/android/app/src/main/AndroidManifest.xml:1]: I can still see Permissions for External Storage, Please remove them

File: example/test/widget_test.dart
Comment (avinashkranjan) [inline example/test/widget_test.dart:1]: Please write test for your code..

**Review by avinashkranjan — CHANGES_REQUESTED** [review]

**Comment by avinashkranjan** [comment]
Hey @Saumya-28 , I have mentioned some changes, also, Write a small function for retriving the captured screenshots.

**Comment by avinashkranjan** [comment]
No, A function where I can get all the SS, so, I can upload them to a server in the background.

**Comment by Saumya-28** [comment]
Hi @avinashkranjan !
I have updated the code, please check and let me know.

**Comment by Saumya-28** [comment]
Hi @avinashkranjan !
Kindly review the code and let me know.

**Comment by avinashkranjan** [comment]
@Saumya-28, Caught up in something, will be reviewed by weekend.

**Comment by Saumya-28** [comment]
Hi @avinashkranjan could you please review the code.

**Review by avinashkranjan — APPROVED** [review]
LGTM..💯

**Pull request merged**